core/src/main/scala/io/chrisdavenport/npmpackage/sbtplugin/NpmPackagePlugin.scala

```diff
@@ -172,11 +172,15 @@ object NpmPackagePlugin extends AutoPlugin {
     val npmPackageType: SettingKey[String] = 
       settingKey("The type of the package - defaults to 'commonjs' for ModuleKind.CommonJSModule or ModuleKind.NoModule, and 'module' for ModuleKind.ESModule")
 
+    lazy val npmPackageExtraFiles: SettingKey[Seq[File]] =
+      settingKey[Seq[File]]("Extra files to copy to the NPM install directory")
```

Comment:
It's copying to `npmPackageOutputDirectory` right, so should we say the package output directory?

```suggestion
      settingKey[Seq[File]]("Extra files to copy to the NPM install directory")
```

Reply:
Done